Classes/common/HTTP/CDTURLSessionInterceptorContext.m

```diff
@@ -0,0 +1,35 @@
+//
+//  CDTURLSEssionFilterContext.m
+//  
+//
+//  Created by Rhys Short on 17/08/2015.
+//
+//
+
+#import "CDTURLSEssionInterceptorContext.h"
+
+@implementation CDTURLSessionInterceptorContext
+
+//turn off the nonnull warning, we are passing null to cause a failure,
```

Comment:
what do you mean by "we are passing null to cause a failure"? I don't understand what you're trying to do here, can you explain?